cypress/support/utils/uiCalendar_CalendarUtils.js

```diff
@@ -0,0 +1,519 @@
+/**
+ * utility functions for ui-calendar automated UI testing
+ */
+
+import {
+  dateFromHHMM,
+  dateFromYYYYMMDD,
+  dateFromYYYYMMDDAndHHMM,
+  dateToTimeOnly,
+  getLocalizedDate, getLocalizedTime,
+  getRelativeDateProximity,
+  dateFromDateAndHHMM
+} from './uiCalendar_DateUtils';
+
+import { WEEKDAYS, WEEKDAY_INDEX, getRelativeWeekdayStatus, weekdayIsBetween } from './uiCalendar_WeekdayUtils';
+
+/** Get all openings and exceptions which apply to this date */
+export function getDateMatches(
```

Comment:
move calendar utils in
cypress/support/utils/

Reply:
Can you clarify the exact file I should move it to? @MaksimTerekhov

Reply:
you can create new file or move your file [uiCalendar_CalendarUtils.js](https://github.com/folio-org/stripes-testing/pull/700/files/6424fb8d2158e26f59a625ba1984b9acfba714de#diff-e64c42dd7f587c476e84d5886b314f9d7591f0e7d3b9bedaa5946cf947ef9fa2) inside folder cypress/support/utils/
this is up to you

Reply:
Oh, I'm a bit lost because uiCalendar_CalendarUtils.js is already in cypress/support/utils